tests/test_init.py

```
@@ -579,6 +581,40 @@ async def test_generate_support_bundle_service_accepts_setup_retry_device(
         assert kwargs["coordinator"] is None
         mock_save_support_bundle.assert_called_once()
 
+    async def test_generate_support_bundle_service_rejects_multiple_device_targets(
+        self,
+        hass: HomeAssistant,
+        mock_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.15.10)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)